README.md

```diff
@@ -79,16 +89,6 @@ command: `FlutterGPT Refactor: Fix Errors`
 
 Please Note: Using OpenAI APIs will incur charges. From our observations, running 500 create widget from description costs only 1$.
 
-## FAQs
```

Comment:
@tanmoy27112000 Did you take the latest changes from `main`. There is some readme content that's being deleted.

Reply:
updated the branch